lib/node_modules/@stdlib/math/base/special/sincos/benchmark/benchmark.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pkg = require( './../package.json' ).name;
+var sin = require( '@stdlib/math/base/special/sin' );
+var cos = require( '@stdlib/math/base/special/cos' );
+var sincos = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var x;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*20.0 ) - 10.0;
+		y = sincos( x );
+		if ( isnan( y[0] ) || isnan( y[1] ) ) {
+			b.fail( 'should not return NaN' );
+		}
+	}
+	b.toc();
+	if ( isnan( y ) ) {
+		b.fail( 'should not return NaN' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
+});
+
+bench( pkg+'::separate-evaluation', function benchmark( b ) {
+	var x;
+	var y;
+	var i;
+
+	y = [ 0.0, 0.0 ];
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*20.0 ) - 10.0;
+		y = [ sin( x ), cos( x ) ];
+		if ( isnan( y[0] ) || isnan( y[1] ) ) {
+			b.fail( 'should not return NaN' );
+		}
+	}
+	b.toc();
+	if ( isnan( y ) ) {
+		b.fail( 'should not return NaN' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
+});
+
+bench( pkg+'::in-place', function benchmark( b ) {
+	var x;
+	var y;
+	var i;
+
+	y = [ 0.0, 0.0 ];
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*20.0 ) - 10.0;
+		sincos( y, x );
+		if ( isnan( y[0] ) || isnan( y[1] ) ) {
+			b.fail( 'should not return NaN' );
+		}
+	}
+	b.toc();
+	if ( isnan( y ) ) {
+		b.fail( 'should not return NaN' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
+});
+
+bench( pkg+'::separate-in-place-evaluation', function benchmark( b ) {
```

Comment:
`separate-evaluation,in-place`. This is not yet documented, but we allow specifying multiple namespaces by delineating each namespace with a comma. This allows us to group here, e.g., the `in-place` benchmarks together and the `separate-evaluation` benchmarks, etc.